test/test_mi_ranking.py

```diff
@@ -157,6 +180,65 @@ def test_unique_categorical_removed(self) -> None:
         self.assertListEqual(list(result_dataframe['idx']), [2, 5, 4, 3])
         self.assertListEqual(list(result_dataframe['name']), ['bravo', 'echo', 'delta', 'charlie'])
 
+    # def test_acled(self) -> None:
```

Comment:
Did you mean to add this in and leave it all commented out?

Reply:
I did, just because if I want to test on a full dataset in the future, I want to be able to just uncomment this.

I can take it out and paste the code in some other file if you want.

Reply:
No, that's fine.  Just wanted to make sure.